saleor/plugins/manager.py

```diff
@@ -224,6 +224,7 @@ def __run_method_on_single_plugin(
         plugin_method = getattr(plugin, method_name, NotImplemented)
         if plugin_method == NotImplemented:
             return previous_value
+        plugin.allow_replica = self._allow_replica
```

Comment:
Why this needs to be provided here? Allow replica is provided here https://github.com/saleor/saleor/blob/f3e720d80af907bc7a0a84c4df8dd926d4989125/saleor/plugins/manager.py#L98